### PR TITLE
[rbdl-orb] Update port to version 3.1.2

### PR DESCRIPTION
**Pull request by ju6ge**
This PR, updates the rbdl-orb port to version 3.1.2. 

What is new:
- adds FindRBDL.cmake files
- updates to the URDF Parser library

### CODE REVIEW

File: ports/rbdl-orb/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "rbdl-orb",
-  "version": "3.0.0",
-  "port-version": 1,
+  "version": "3.1.2",
+  "port-version": 2,
```
Comment (JonLiu1993) [inline ports/rbdl-orb/vcpkg.json:4]:
```suggestion
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

File: versions/r-/rbdl-orb.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+    {
+      "git-tree": "c6cec01f39c02f0ac2164e31e4615e21a80f4515",
+      "version": "3.1.2",
+      "port-version": 0
+    },
+    {
+      "git-tree": "d47e83ceec5a9f75cd0fef6519851f68e6b8d15a",
+      "version": "3.1.2",
+      "port-version": 2
+    },
```
Comment (BillyONeal) [inline versions/r-/rbdl-orb.json:12]:
```suggestion
```

Deleting this nonexistent version from the database will allow us to merge this. I tried to do it for you but the PR was created without permissions for maintainers to fix.

**Review by BillyONeal — CHANGES_REQUESTED** [review]

File: versions/r-/rbdl-orb.json (hunk 2 above)
Comment (ju6ge) [reply versions/r-/rbdl-orb.json:12]: sry next time i will allow maintainer edits ;)

File: versions/r-/rbdl-orb.json [hunk 3]
```diff
@@ -1,5 +1,11 @@
 {
   "versions": [
+    {
+      "git-tree": "c6cec01f39c02f0ac2164e31e4615e21a80f4515",
+      "version": "3.1.2",
+      "port-version": 0
+    },
+    {
```
Comment (BillyONeal) [inline versions/r-/rbdl-orb.json:8]: I'm not sure what happened here. The way I normally fix this is something like:

```
git fetch origin #assuming that microsoft/vcpkg is called origin
git merge origin/master
git checkout origin/master -- versions # this removes all edits to the version database
vcpkg x-add-version rbdl-orb
git commit -am "Update version database"
git push
```

Comment (PhoebeHui) [reply versions/r-/rbdl-orb.json:8]: There has extra '{' here.☺

Comment (PhoebeHui) [reply versions/r-/rbdl-orb.json:8]: I have no permission to push the changes to this PR, @ju6ge, could you fix this issue?

Comment (PhoebeHui) [inline versions/r-/rbdl-orb.json:8]:
```suggestion
```

Comment (ju6ge) [reply versions/r-/rbdl-orb.json:8]: sure ;)

**Review by PhoebeHui — APPROVED** [review]
Thanks for your quick fix!

**Pull request merged**